README.md

```diff
@@ -11,123 +11,94 @@ This repository contains the `tng-sdk-validation` component that is part of the
 
 `tng-sdk-validation` can be used through the CLI, as service  or as a micro-service running inside a docker container.
 
-## Dependencies
-
-* This tool has been designed to be executed in Linux system and Python 3.6 or higher.
-
-* It is necessary to install [tng-sdk-project](https://github.com/sonata-nfv/tng-sdk-project) first as *tng-sdk-validation* depends on it. Please follow the instructions provided in the Readme file to install it.
+# Installation and Dependencies
 
+This tool has been designed to be executed in Linux system and Python 3.6 or higher. It's necessary to install [tng-sdk-project](https://github.com/sonata-nfv/tng-sdk-project) first because first as *tng-sdk-validation* depends on it. Moreover, other requirements of the tool are [requirements](https://github.com/MiguelRivasQuobis/tng-sdk-validation/blob/master/requirements.txt).
+You can install it with the following statement.
+```
+pip3 install git -r requirements.txt
+```
 
-## Installation
+## Automated:
 
 It is possible use the following command for automatic installation
 ```
 pip3 install git+https://github.com/sonata-nfv/tng-sdk-validation.git
 ```
-Alternatively:
-1. Clone the master branch of this repository and access the directory.
+## Manual:
 
-2. Then install the Python dependencies defined in requierements file.
-```bash
-$ pip3 install -r requirements.txt
 ```
-3. The last step is to install *tng-sdk-validation* tool.
-```bash
-$ python3 setup.py install
+git clone git@github.com:sonata-nfv/tng-sdk-validate.git
+cd tng-sdk-validate
+sudo python3 setup install
 ```
-4. Once installed, the command `tng-sdk-validate` should be available in your Linux system.
 
-## Usage
+# Usage
+
+The packager can either be used as command line tool (CLI mode) or deployed as a micro service which offers a REST API
+
+## CLI mode
 
-### CLI mode
+Runs the validator locally from the command line. Details about all possible parameters can be shown using:
+
+```
+tng-sdk-validate -h
+```
+### Validation
 
 The CLI interface is designed for developer usage, allowing to quickly validate SDK projects descriptors, package descriptors, service descriptors and function descriptors. The different levels of validation are syntax, integrity, topology and custom_rules. They can only be used in the following combinations:
```

Comment:
custom_rules? remove _

Reply:
there are service descriptors and function descriptors. also test descriptors, sla, slices... but no project descriptors nor package descriptors. There are projects and packages with test/service/etc descriptors inside.

Reply:
this is information for the wiki, not for the README. README must be short and direct. No explanations this long must be given.